x-pack/plugin/vector-tile/src/yamlRestTest/resources/rest-api-spec/test/10_basic.yml

```diff
@@ -167,3 +167,27 @@ setup:
         zoom: 0
         body:
           sort: []
+
```

Comment:
Thanks for adding test cases for `track_total_hits` in the request body as well.